ext/hash/tests/hash_file_error.phpt

```diff
@@ -36,9 +40,7 @@ unlink( $filename );
 *** Testing hash_file() : error conditions ***
 
 -- Testing hash_file() function with an unknown algorithm --
-
-Warning: hash_file(): Unknown hashing algorithm: %s in %s on line %d
-bool(false)
+hash_file(): Argument #1 ($algo) must be a valid hashing algorithm
```

Comment:
I think we might want to include `, "%s" given` in this error message, as the information was present before.

Reply:
Only for `hash_file()`, right? I got rid of `%s` from `hash()` because sensitive data might leak if the user accidentally swaps the argument order. I'm not sure the same can happen with a file path though. What do you think?

Reply:
Hm, good point. In that case let's leave it as is.